framework/db/pgsql/Schema.php

```diff
@@ -213,6 +217,50 @@ protected function findTableNames($schema = '')
     }
 
     /**
+     * Returns all views names in the database.
+     * @param string $schema the schema of the views. Defaults to empty string, meaning the current or default schema.
+     * @return array all views names in the database. The names have NO schema name prefix.
```

Comment:
Oh, great, actually I don't know how to change a commit from a pull request :/ . May you please do it yourself? Sorry and thanks!